kotlin/src/test/kotlin/org/axonframework/extensions/kotlin/CombiningCommandCallbackTest.kt

```diff
@@ -0,0 +1,102 @@
+/*-
+ * #%L
+ * Axon Framework - Kotlin Extension
+ * %%
+ * Copyright (C) 2019 AxonIQ
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ * #L%
+ */
+package org.axonframework.extensions.kotlin
+
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.verify
+import org.axonframework.commandhandling.CommandMessage
+import org.axonframework.commandhandling.GenericCommandMessage
+import org.axonframework.commandhandling.GenericCommandResultMessage
+import org.axonframework.commandhandling.distributed.CommandDispatchException
+import org.axonframework.messaging.MetaData
+import org.junit.Test
+import java.util.*
+import kotlin.test.fail
+
+class CombiningCommandCallbackTest {
+    private val command = ExampleCommand("1")
+    private val commandMessage: CommandMessage<ExampleCommand> = GenericCommandMessage.asCommandMessage(command)
+    private val responsePayloadUUID = UUID.randomUUID().toString()
+    private val commandResultMessage = GenericCommandResultMessage.asCommandResultMessage<String>(responsePayloadUUID)
+    private val exceptionalCommandResultMessage = GenericCommandResultMessage.asCommandResultMessage<String>(CommandDispatchException("Exception message"))
+    private val metaData = MetaData.with("key", "value")
+
+    @Test
```

Comment:
Should've told you earlier (so sorry for this), but the framework has moved to JUnit 5 entirely at the moment and the majority of the Kafka Extension has been adjusted. Might be beneficial take this stance for the Kotlin extension too.